### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by anderCM**
- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces eg.

```
decode("-- -.--   -. .- -- .")
```

- Reuse methods whenever possible. Make sure that your code is DRY.

- Once your code is ready use it to decode the message from the old bottle:

```
.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...
```

### CODE REVIEW

File: decoder.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  words = code.split('   ')
+  words.each do |word|
+    characters = word.split
+    characters.each do |character|
+      print morse_dict[character]
+    end
+    print ' '
+  end
+end
+decode('-- -.--   -. .- -- .')
```
Comment (bobb-Rob) [inline decoder.rb]: - [x] Based on the project requirements, you are expected to have at least three methods with a single responsibility  - e.g 
    - `decode_char` - responsibility is to decode a single character. 
    - `decode_word` - responsibility is to decode a word and 
    - `decode` - responsibility is to decode a sentence. 
Then, use the **character decode method** inside the **word decode method** and so on.

_**Please note that the names of the methods can be different from what I call them above but each name should be meaningful at first glance.**_ 


Project requirement 👇🏾 
---- |
image


Please carefully read the requirement again and implement this project accordingly.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @anderCM, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: decoder.rb (hunk 1 above)
Comment (anderCM) [reply decoder.rb]: Thanks for sharing this information, I will create the `decode_char`, `decode_word` and `decode` methods

**Pull request merged**